docs/development/maintenance.md

```diff
@@ -13,104 +15,124 @@ modules of ILIAS.
-	Only relevant if **the maintenance_model** is set to "Coordinator".
-* **"[pr_management](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#additional-competences)"** : 
-    An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"[issue_management](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/maintenance.md#additional-competences)"** :
-  An array in the form [ `<username> (<userid>)` ] pointing to valid users on https://docu.ilias.de.
-* **"tester"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
-* **"testcase_writer"**: One entry in the form `<username> (<userid>)` pointing to a valid user on
-  https://docu.ilias.de.
+# Authorities
+The ILIAS-Community strives to create and maintain a secure, reliable, and
+adaptable learning management. We foster participation by a diverse set of
+developers, designers, testers and other contributors, but we also have to guarantee the sustainability and the quality
+of the ILIAS source code.
+
+To make sure people with diverse backgrounds and capabilities can participate
+in our Community and contribute to the development of ILIAS and its code base,
+we split the code into units (often called components, even though the term
+is hard to define) and we define a set of authorities community members can have
+concerning these units of code. We understand an authority as the counterpart of
+a responsibility: The people having the authorities to do something in a unit of
+code also assume the responsibility for the corresponding functions.
+
+For the context of ILIAS, we define **four** different authorities:
+
+1. **Authority to Sign of Code Changes**
+2. **Authority to Sign of Conceptual Changes**
+3. **Authority to Sign of Testcase Changes**
+4. **Authority to (De-)Assign Authorities**
+
+Each of these authorities can be held by a different set of people. This means
+that the social organisation of different groups working on different parts of
+the Code of ILIAS can be different.
+Right now ILIAS knows a few different social structures for the maintenance of
+units in the code of ILIAS:
+
+* In the **"Classic Model"** all authorities are concentrated in one person and
+this person works mostly alone.
+* In the **Coordinator Model** all authorities are concentrated in one or more
+people and they work together with other developers in the community to improve
+the code.
+* In the **"Test and Assessment Model"** the authorities **to Sign of Conceptual
+Changes**, **to Sign of Testcase Changes**, and **to (De-)Assign Authorities**
+lies with one person and the **Authority to Sign of Code Changes** with two
+others.
+
+More will surely emerge as the optimal solution for each unit is found.
+
+# Responsibilites
+Independently of the social organisation for each unit of code the following
```

Comment:
In dieser Zeile sollte ein Komma ergänzt werden – auch nach mehrmaligem Lesen bin ich nicht zu 100 % sicher, welcher Bezug genau gemeint (und der bessere) ist. Entweder:

> Independently of the social organisation, for each unit of code the following

oder:

> Independently of the social organisation for each unit of code, the following

Reply:
Kommt eigentlich nicht wirklich darauf an, wobei die erste Variante, meiner Meinung nach, deutlicher ist.